docs/core/install/linux-rhel.md

```diff
@@ -1,33 +1,35 @@
 ---
-title: Install .NET on RHEL - .NET
+title: Install .NET on RHEL and CentOS Stream
 description: Demonstrates the various ways to install .NET SDK and .NET Runtime on Red Hat Enterprise Linux.
 author: adegeo
 ms.author: adegeo
-ms.date: 11/05/2021
+ms.date: 03/21/2022
 ---
 
-# Install the .NET SDK or the .NET Runtime on RHEL
+# Install the .NET SDK or the .NET Runtime on RHEL and CentOS Stream
 
 .NET is supported on Red Hat Enterprise Linux (RHEL). This article describes how to install .NET on RHEL.
 
 [!INCLUDE [linux-intro-sdk-vs-runtime](includes/linux-intro-sdk-vs-runtime.md)]
 
 ## Register your Red Hat subscription
 
-To install .NET from Red Hat on RHEL, you first need to register using the Red Hat Subscription Manager. If this hasn't been done on your system, or if you're unsure, see the [Red Hat Product Documentation for .NET](https://access.redhat.com/documentation/en-us/net/5.0).
+To install .NET from Red Hat on RHEL, you first need to register using the Red Hat Subscription Manager. If this hasn't been done on your system, or if you're unsure, see the [Red Hat Product Documentation for .NET](https://access.redhat.com/documentation/en-us/net/6.0).
```

Comment:
Is it worth calling out that this step is not needed on CentOS Stream? Or is that going too much into the particulars of that distro?